anomaly-event-processor/src/main/java/org/hypertrace/alert/engine/anomaly/event/processor/AnomalyEventProcessor.java

```diff
@@ -0,0 +1,54 @@
+package org.hypertrace.alert.engine.anomaly.event.processor;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import org.hypertrace.alert.engine.anomaly.event.processor.notification.WebhookNotifier;
+import org.hypertrace.alert.engine.metric.anomaly.datamodel.ActionEvent;
+import org.hypertrace.alert.engine.metric.anomaly.datamodel.EventRecord;
+import org.hypertrace.alert.engine.metric.anomaly.datamodel.MetricAnomalyViolation;
+import org.hypertrace.alert.engine.notification.transport.webhook.WebhookSender;
+import org.hypertrace.alert.engine.notification.transport.webhook.http.HttpWithJsonSender;
+
+class AnomalyEventProcessor {
```

Comment:
Would rename this to NotificationEventProcessor or ActionEventProcessor. AnomalyEvent will be consume by decision engine.

Reply:
Done